adc/ad7779/ad7779.h

```diff
@@ -89,4 +89,58 @@ int ad7779_get_status(uint8_t *status_buf);
 /* For debugging purposes */
 int ad7779_print_status(void);
 
+/* Logging */
+#define COL_RED    "\033[1;31m"
+#define COL_CYAN   "\033[1;36m"
+#define COL_NORMAL "\033[0m"
+
+#define LOG_TAG "ad7779: "
+
+#ifdef NDEBUG
+#define log_debug(fmt, ...)
+#else
+#define log_debug(fmt, ...) \
```

Comment:
I think we could leave these macros as one-liners (and optionally surround them with `/* clang-format off[/on] */`